sysutils/pv/Portfile

```
@@ -21,43 +20,22 @@ long_description    pv - Pipe Viewer - is a terminal-based tool for monitoring \
                     through, how long it has taken, how near to completion it \
                     is, and an estimate of how long it will be until completion.
 
-homepage            http://www.ivarch.com/programs/pv.shtml
+homepage            https://www.ivarch.com/programs/pv.shtml
+master_sites        ${codeberg.homepage}/releases/download/v${version}
 
-checksums \
-    size    116119 \
-    rmd160  da9786dd633225ec0323c25a073eb7c05eaf6159 \
-    sha256  bb7394a6c3a8240c3dafa8507c2e74df7164260854b0ddec89e1b2d2b8bce5e1
+distname            ${name}-${version}
+checksums           rmd160  8ff4da8318795e8fcfc9a474fb9c9c0d8988aef8 \
+                    sha256  d22948d06be06a5be37336318de540a2215be10ab0163f8cd23a20149647b780 \
+                    size    327072
 
-configure.args      --mandir=${prefix}/share/man \
-                    --infodir=${prefix}/share/info
+patch.pre_args      -p1
+patchfiles          fix-modifiers-direct-io-test.diff
 
-test.run            yes
-
-depends_lib-append      port:gettext
-depends_build-append    port:autoconf
+depends_lib-append  port:gettext
```

`gettext` is split into `gettext` and `gettext-runtime`, so I think this should instead be:

```suggestion
depends_build-append \
                    port:gettext
depends_lib-append  port:gettext-runtime
```

---

Background: https://lists.macports.org/pipermail/macports-dev/2021-December/044041.html